src/Nox.Reference.VatNumbers/Services/IVatValidationService.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Nox.Reference.VatNumbers.Constants;
+using Nox.Reference.VatNumbers.Models;
+using System.Text.RegularExpressions;
+
+namespace Nox.Reference.VatNumbers.Services
+{
+    internal interface IVatValidationService
+    {
+        public ValidationResult ValidateVatNumber(IVatNumber vatNumber);
+
+        public static void ValidateRegex(
```

Comment:
is there any purpose of introducing static method in interface? Whether base class is better than static?

Reply:
Moved to base class